scripts/provisioning/s3_setup

```diff
@@ -281,31 +241,18 @@ then
     # validate pre-requisites before proceeding
     # we already have 'exit' statement in '_s3_setup' binary at failure scenario,
     # no need to add 'exit' here.
-    echo "INFO: validating pre-requisites for provisioning s3server.."
-    $install_dir/_s3_setup post_install
-
-    # Fetch and update cluster id
-    # To decrypt openldap passwords, we need 'cluster_id' updated in s3_cluster.yaml
-    echo "INFO: Fetching and updating cluster id in s3_cluster.yaml .."
-    fetch_update_cluster_id
-
-    # get openldap password from confstore
-    echo "INFO: Get openldap passwords from confstore.."
-    get_ldap_password
 
+    # NOTE: Call to create_auth_jks_password will be removed later
     # Create random password for auth jks keystore
     echo "INFO: Creating random jks password for Auth server.."
     create_auth_jks_password
```

Comment:
ToBe evaluated

Reply:
Create Cleanup JIRA

Reply:
Should I create this ticket ?

Reply:
@sachitanands  please create a ticket for this.

Reply:
@pratyush-seagate , @nileshgovande created ticket : https://jts.seagate.com/browse/EOS-18331 for this JKS related work